src/vue/components/panel-compact.vue

```diff
@@ -268,107 +323,80 @@ export default {
     infoIcon: InfoIcon,
     grammarIcon: GrammarIcon,
     treebankIcon: TreebankIcon,
+    wordUsageIcon: WordUsageIcon,
     alphTooltip: Tooltip,
     lookup: Lookup,
-    reskinFontColor: ReskinFontColor
+    reskinFontColor: ReskinFontColor,
+    wordUsageExamplesBlock: WordUsageExamplesBlock
   },
   directives: {
     onClickaway: onClickaway
   },
+  positionClassVariants: {
```

Comment:
why are positionClassVariants, minWidth and defaultScrollPadding being set as $options here and not properties?

Reply:
Those are, in our case, constants. They're used in one place only, when we add a position class when a position is switched. Having them in `data` would make them reactive, which we don't need. Moving them to custom options prevents Vue from attaching reactivity monitoring to them.

They could be constants outside of Vue config object, or they could be defined in the UI object of UI controller, or we could define them in a method that uses them. Not sure what's the best way to handle this. It would be ideal to gather all class-related data together, but at the same time this is panel-specific, so it should probably be within the panel. What do you think?

Reply:
ok, I suspected it was to do with reactivity. I think it's okay as-is, please just add a comment to the code to explain this decision.

Reply:
OK